### PR TITLE
Migrates the hellspawn stuff over from hackmd

### PR DESCRIPTION
**Pull request by LemonInTheDark**
## About The Pull Request

Migrates some content discussing byond pain points over from hackmd, this should make it more visible, which I think is a good thing.

In other news, we really should reorg this document, should make subdocs like oranges wanted

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -777,6 +777,42 @@ Isn't that confusing?
 
 There is also an undocumented keyword called `static` that has the same behaviour as global but more correctly describes BYOND's behaviour. Therefore, we always use static instead of global where we need it, as it reduces suprise when reading BYOND code.
 
+### Byond Hellspawn 
+
+Put stuff that shouldn’t work but does, or should work but doesn’t here so we don’t forget about it.
+#### \
+
+\ tells the compiler to read the next line as if it was one of our own. As an example:
```
Comment (Mothblocks) [inline .github/CONTRIBUTING.md:785]: do we need to put this in contributing.md? dreamchecker picks it up

Comment (LemonInTheDark) [reply .github/CONTRIBUTING.md:785]: ah we have a lint for this now? Great!

Comment (Mothblocks) [reply .github/CONTRIBUTING.md:785]: ya

Comment (Kylerace) [reply .github/CONTRIBUTING.md]: haha yeah but who could benefit from that I wouldn't know them 😳

**Pull request merged**